2832 - Add a fix datepicker destroy and mask invocation (#2837)
infor-design/enterprise: docs/CHANGELOG.md

- `[Datagrid]` Made the text all white on the targeted achievement formatter. ([#2730](https://github.com/infor-design/enterprise/issues/2730))
- `[Datagrid]` Fixed keyword search so that it will again work with client side paging. ([#2797](https://github.com/infor-design/enterprise/issues/2797))
- `[Datepicker]` Moved the today button to the datepicker header and adding a setting to hide it if wanted. ([#2704](https://github.com/infor-design/enterprise/issues/2704))

> - `[Datepicker]` Fixed a bug in datepicker that the destroy would not re-add masking functionality. [2832](https://github.com/infor-design/enterprise/issues/2832))

Reviewer: ```suggestion
- `[Datepicker]` Fixed a bug in datepicker where the destroy method does not readd the masking functionality. [2832](https://github.com/infor-design/enterprise/issues/2832))
```
